docs/_main/getting_started.md

```diff
@@ -27,8 +27,8 @@ Some are broad and authorize your app to access lots of data, while others are v
 a tiny sliver. Your users (and their IT admins) will have opinions about which data your app should access, and only
 agree to install the app if the data permissions seem reasonable, so we recommend finding the scope(s) with the least
 amount of privilege for your app's needs. In this guide we will use the Web API to post a message. The scope required
-for this is called [`chat:write`](https://api.slack.com/scopes/chat:write). Use the dropdown under the "Bot Token Scopes" header and add the scope, then click
-"Save Changes".
+for this is called [`chat:write`](https://api.slack.com/scopes/chat:write). Click "Add an OAuth Scope", find the
```

Comment:
I think it's important to keep the line about _Use the dropdown under the "Bot Token Scopes" header_ (or some reference as to which section to use) because if that section/accordion is toggled closed, the **Add an OAuth Scope** button is hidden. 

Plus, there is an identically titled button under the **User Token Scopes** section.

Reply:
Oooo good catch. I've tweaked the wording.